app/controllers/repos_controller.rb

```diff
@@ -62,11 +62,9 @@ def fetch_image_ids
       else
         ids = legacy_image_list(repo)
       end
-
-      ids.map! { |id| id.gsub(ENV['IIIF_IMAGE_SERVER'], '').gsub(/^[^=]*=/, '').gsub('/info.json', '') }
```

Comment:
Just wanted to note that this logic has been moved into Marmite via this PR: https://github.com/upenn-libraries/marmite/pull/29